Dockerfile_Jellyfin-Intro-Scanner

```diff
@@ -0,0 +1,10 @@
+FROM python:3.8
+COPY . /app
+RUN apt-get update
+RUN apt-get install -y ffmpeg libsm6 libxext6
+RUN pip install --no-cache-dir -r /app/requirements.txt
+WORKDIR /app
+ENTRYPOINT ["python"]
+CMD [ "jellyfin.py", "-d", "-j" ]
```

Comment:
I just added logging to file. with `-l` a new log is created for each session inside `/config/logs`
eg: `log_2022_03_30-10_50_59_PM.txt` 

```suggestion
CMD [ "jellyfin.py", "-d", "-l", "-j" ]
```